test/gateway/api_mock_data.dart

```diff
@@ -0,0 +1,6 @@
+class MockData {
+  dynamic req;
+  dynamic res;
```

Comment:
Can this type be specific? Or at least reduce the scope of what it can be.

Reply:
It can be Map<String, dynamic>, but will require every model has toJson method